### PR TITLE
Add plugins for Groovy and Grails

### PR DESCRIPTION
**Pull request by weibemoura**

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 fails=0
 for i in $(grep -o 'travis.*\.svg' README.md); do
-  url=$(echo $i | sed "s/\//\/repositories\//" | sed "s/svg/json/")
-  curl -sq https://api.$url | grep 'last_build_result":0' > /dev/null
+  url=$(echo $i | sed "s/\//\/repositories\//" | sed "s/\.svg//")
+  curl -sq -H "Accept: application/json" https://api.$url | grep 'last_build_result":0' > /dev/null
```
Comment (Stratus3D) [inline test.sh]: Can you explain what these changes do? We do have some failing plugins, but I was not aware of any issues in this code.

File: plugins/groovy [hunk 2]
```diff
@@ -0,0 +1 @@
+repository = https://github.com/weibemoura/asdf-groovy
```
Comment (Stratus3D) [inline plugins/groovy]: Can you add `.git` to the end of these URLs? The actual Git repo is `https://github.com/weibemoura/asdf-groovy.git`

File: README.md [hunk 3]
```diff
@@ -54,6 +54,8 @@ place for people (and asdf itself) to look for plugins.
 | GoHugo    | [mgladdish/asdf-gohugo](https://bitbucket.org/mgladdish/asdf-gohugo) | [![Build Status](https://img.shields.io/bitbucket/pipelines/mgladdish/asdf-gohugo.svg)](https://bitbucket.org/mgladdish/asdf-gohugo/addon/pipelines/home)
 | GraalVM    | [vic/asdf-graalvm](https://github.com/vic/asdf-graalvm) | [![Build Status](https://travis-ci.org/vic/asdf-graalvm.svg?branch=master)](https://travis-ci.org/vic/asdf-graalvm)
 | Gradle    | [rfrancis/asdf-gradle](https://github.com/rfrancis/asdf-gradle) | [![Build Status](https://travis-ci.org/rfrancis/asdf-gradle.svg?branch=master)](https://travis-ci.org/rfrancis/asdf-gradle)
+| Groovy    | [weibemoura/asdf-groovy](https://github.com/weibemoura/asdf-groovy) | [![Build Status](https://travis-ci.org/weibemoura/asdf-groovy.svg?branch=master)](https://travis-ci.org/weibemoura/asdf-groovy)
+| Grails    | [weibemoura/asdf-grails](https://github.com/weibemoura/asdf-grails) | [![Build Status](https://travis-ci.org/weibemoura/asdf-grails.svg?branch=master)](https://travis-ci.org/weibemoura/asdf-grails)
```
Comment (Stratus3D) [inline README.md]: The plugins need to be listed in alphabetical order in this table.

**Comment by Stratus3D** [comment]
Both plugins themselves look good.

File: test.sh (hunk 1 above)
Comment (weibemoura) [reply test.sh]: sorry those changes were a misnomer

File: plugins/groovy (hunk 2 above)
Comment (weibemoura) [reply plugins/groovy]: I sent the changes to a new commit

File: README.md (hunk 3 above)
Comment (weibemoura) [reply README.md]: I sent the changes to a new commit

**Comment by Stratus3D** [comment]
Thanks for the PR!

**Pull request merged**